### PR TITLE
Gateway class needs to be namespaced with Omnipay

### PR DESCRIPTION
**Pull request by elliot-sawyer**

### CODE REVIEW

**Comment by elliot-sawyer** [comment]
The reason for this lies with the method `Helper::getGatewayClassName()`:

`return '\\Omnipay\\'.$shortName.'Gateway';`

File: composer.json [hunk 1]
```diff
@@ -25,13 +25,14 @@
...
-            "Dcentrica\\Omnipay\\BTCPayServer\\": "src/"
+            "Omnipay\\BTCPayServer\\": "src/"
         }
     },
     "require": {
         "php": ">=7.1",
         "omnipay/common": "~3.0",
-        "bitpay/php-client": "dev-master"
+        "bitpay/php-client": "dev-master",
+        "btcpayserver/btcpayserver-php-client": "~1.0.0"
```
Comment (phptek) [inline composer.json:35]: We'll stick with `bitpay/php-client` - `btcpayserver/btcpayserver-php-client` looks to be buggy as hell.

File: src/Gateway.php [hunk 2]
```diff
@@ -6,7 +6,7 @@
  * @author  Elliot Sawyer <elliot.sawyer@gmail.com>
  */
 
-namespace Dcentrica\Omnipay\BTCPayServer;
+namespace Omnipay\BTCPayServer;
```
Comment (phptek) [inline src/Gateway.php:9]: Patched this separately.

File: composer.json [hunk 3]
```diff
@@ -25,13 +25,14 @@
     ],
     "autoload": {
         "psr-4": {
-            "Dcentrica\\Omnipay\\BTCPayServer\\": "src/"
+            "Omnipay\\BTCPayServer\\": "src/"
```
Comment (phptek) [inline composer.json:28]: Patched this separately.

**Pull request closed without merging**